common/app/controllers/EmailSignupController.scala

```diff
@@ -42,6 +42,7 @@ object ListIds {
   val greenLight = 28
   val povertyMatters = 113
   val theLongRead = 3322
+  val weekendReading = 3743
```

Comment:
These are the ListIDs (primary key) for the emails in ExactTarget (our email marketing software). Editorial emails are created in ExactTarget by Editorial, who then inform their contact in Digital of the listID, so that it can be added to [`EmailSubscriptions.scala`](https://github.com/guardian/frontend/blob/master/common/app/model/EmailSubscriptions.scala) and appear in the email-prefs centre.

(I would really love if these listIDs weren't also coded into `EmailSignupController.scala` and that there was one, canonical source for all email data)